### PR TITLE
Add usage example using YAML syntax

### PR DESCRIPTION
**Pull request by khakiout**
Add example using YAML Syntax.

In reference for issue #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,36 @@ This Action wraps the [Serverless Framework](https://serverless.com) to enable c
...
+      uses: actions/npm@1.0.0
+      with:
+        args: install
+
+    - name: serverless deploy
+      uses: serverless/github-action@master
+      with:
+        args: deploy
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_SECRET_KEY_ID }}
```
Comment (Gr8z) [inline README.md]: You have a typo here, it should be ${{ secrets.AWS_ACCESS_KEY_ID }}

Comment (khakiout) [reply README.md]: Will fix! Thanks for the heads up

Comment (khakiout) [reply README.md]: Done.

**Comment by dschep** [comment]
no real need to keep the old way and i idn't see this pr before making the change myself

**Pull request closed without merging**